agents/check_mk_agent.macosx

```diff
@@ -165,7 +166,7 @@ echo Version: 1.5.0p20
 echo AgentOS: macosx $osver
 
 echo '<<<df>>>'
-df -kPT hfs,apfs | egrep -v "Time Machine|com.apple.TimeMachine.localsnapshots|/Volumes/Recovery|/private/var/vm" | sed 1d | \
+df -kPT hfs,apfs | egrep -v "Time Machine|com.apple.TimeMachine.localsnapshots|/Volumes/|/private/var/vm" | sed 1d | \
```

Comment:
Unrelated to the main issue, but I believe anything mounted as a /Volume you don't want to report (these would be USB drives and mounted disk images from installations--in other words all temporary drives). *fingers crossed*

Reply:
In fact we monitor several external disks below `/Volumes/` so I decided to keep the former expression.